<launch>
    <arg name="hybrid_astar" default="true"/>

    <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(find planning)/map/map005.yaml"/> -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(find hybrid_astar)/maps/map.yaml"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find planning)/rviz/config01.rviz"/>

    <group if="$(arg hybrid_astar)">
        <node pkg="planning" type="hybrid_astar_planning_node" name="hybrid_astar_planning_node" output="screen"/>
        <param name="steering_change_penalty" value="2.0"/>
        <param name="steering_penalty" value="1.5"/>
        <param name="reversing_penalty" value="3.0"/>
        <param name="shot_distance" value="0.5"/>
    </group>

</launch>